src/index.html

```diff
@@ -1,13 +1,54 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img class="card__img"
+           alt="imac"
+           src="images/imac.jpeg">
+
+      <div class="card__link">
+        <a>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</a>
+      </div>
+
+      <div class="card__code">
+        Product code: 195434
+      </div>
+
+      <div class="card__grade">
+        <div class="card__grade-stars">
+          <div class="stars-4">
+            <div class="stars-4__star"></div>
+            <div class="stars-4__star"></div>
+            <div class="stars-4__star"></div>
+            <div class="stars-4__star"></div>
+            <div class="stars-4__star"></div>
+          </div>
+        </div>
+        <div class="card__grade-reviews">
+          Reviews: 5
+        </div>
+      </div>
+
+      <div class="card__price">
+        <div class="card__price-text">
+          <span>Price:</span>
```

Comment:
Sorry, hear i am use "span", because mentor said me i cant right text witout some tags, this quote "Use semantic tags like <p> for text content instead of using only div's" , but with <p> all crash and dont work. I found way "span" and use "p" (and delete for all p margin, pading ...), help me please, how beter?

Reply:
`<span>` it's not semantic :) remove it :)